api/app/signals/apps/questionnaires/rest_framework/views/public.py

```diff
@@ -34,6 +34,11 @@ class PublicQuestionnaireViewSet(DatapuntViewSet):
 
     authentication_classes = ()
 
+    @action(detail=True, url_path=r'graph/visual/?$', methods=['GET', ])
+    def graph(self, *args, **kwargs):
+        questionnaire = self.get_object()
+        return Response(questionnaire.graph.draw())
```

Comment:
I am not sure I would have all the drawing functionality in the Django model for a Graph. It sounds more like a job for a service that could be called in this view.

Reply:
I just copied this from the code that is already in the master branch. If we really want to use a `MultiDiGraph` then we can add it in the next PR/Refactor. I think this is something you are already working on in PR #861 https://github.com/Amsterdam/signals/pull/861/files#diff-10364d2fe9ac0385fe9a77a5e47dd72c73d9286d9037940b6a209f330ab73aceR52

Reply:
This I forgot to remove, it was a test.